docs/cluster-configuration.md

```diff
@@ -828,3 +828,25 @@ Labels
     annotation_name="kubevirt.kubevirt.io/jsonpatch"
     severity=info
 ```
+## Log verbosity
+Currently logging verbosity is only supported for Kubevirt.
+
+### Kubevirt
+In order to define logging verbosity for Kubevirt, the following can be configured on HyperConverged CR:
```

Comment:
As a reader of this documentation, I find it a bit crypt. I would add some more details. For example, what are the options under the `kubevirt` field? should I have to list all of them? What are the numbers? what is the range of them (e.g. 8 seems to be out of range). If you have a link that explains all of it in kubevirt project/api, please add it.

Reply:
I understand your feeling, but these knobs are defined within Kubevirt as a developer configuration - maybe that's why it isn't really documented or well defined (yet).

> what are the options under the kubevirt field?

See what I at the bottom: "All the values defined [here](https://kubevirt.io/api-reference/master/definitions.html#_v1_logverbosity) can be applied". I've avoided re-writing them here intentionally - if they would change in the future this docs would become obsolete.

> should I have to list all of them?

You're not and it's also in the API link I've added - but I can add it explicitly if you insist - although again I think it's problematic for the same reason. This API could change in the future, we better write it in a single place and reference it from other places.

> What are the numbers? what is the range of them (e.g. 8 seems to be out of range)

As I've said, this is not well defined nor documented, and can change in the future. 8 is a complete valid value.

> In addition to the inline comment, please also remove the ##### Higher Log Verbosity chapter in docs/cluster-
configuration.md

Done :+1:

Reply:
I guess I wasn't clear. The questions above are not for the code review, but questions that the readers may ask themselves. I another words, I think we need more verbose documentation.

Reply:
@iholder-redhat ^^^